JitsiConference.js

```diff
@@ -374,6 +374,14 @@ JitsiConference.prototype._init = function(options = {}) {
             });
     this.participantConnectionStatus.init();
 
+    // Add the ability to enable callStats only on a certain percentage of users based
+    // on config.js setting.
+    let disableCallStats = false;
+
+    if (config.testing && config.testing.callStatsUserThreshold) {
+        disableCallStats = Math.random() > config.testing.callStatsUserThreshold;
```

Comment:
This will lead to callstats being enabled for some participants in a conference and not others. Are we okay with that? Can we do this pseudo-randomly based on the conference name instead?

Reply:
How do we do based on the conference name ?